Overview.md

```diff
@@ -29,23 +29,77 @@ page_nav:
+
+### Merging external calendars
+
+Agenda enables simplified [connections to your external scheduling applications](insert link to Connect to a Source tutorial) from the Sources tab in the UI. You can choose from several scheduling applications to [merge your meeting information](insert link to Import a Meeting tutorial) into one space. 
+
+![The Sources user interface with a list of all available source connections to external applications](images/overview-sources-view.png)
+
+Agenda unifies this data to provide you and your team members with a clearer view of each other's availability. 
+
+While Agenda provides an overview of your upcoming schedule from the home screen, meetings scheduled in Agenda can also be pushed to your external scheduling applications to ensure all of your calendars are up to date. This is helpful if you work with external partners or clients who are not set up with Agenda and require access to your availability in a different platform.
+
+## System architecture
+
+Agenda creates a two-way data pipeline that streams real-time data between external calendar applications to ensure that your team's schedules are always aligned.
+
+![System architecture diagram outlining Agenda's connections and core components. The client connects to the business logic layer, which handles connections with external sources and Agenda's data warehouse](images/overview-system-architecture.png)
+
+Agenda connects to several external scheduling applications and meeting platforms to ingest user meeting data. External meeting data is streamed into Agenda for processing and is used to update the meeting data set for a given user profile. Meeting information for each user is stored and used to update availability when scheduling new meetings with teams. The availability of external meeting information from connected sources allows Agenda to suggest the most appropriate meeting times based on each team member's separate schedules.
+
+At a high-level, Agenda is comprised of the following major components:
+
+- **Three main views:** Allows you to view and schedule meetings, connect to sources, and manage your teams.
+
+- **Data collection and business logic layer:** Where all of the disparate meeting data for users is combined from external sources to provide a singular view of each user's availability. This is where meeting requests are processed, updated, and made available for export back to your connected sources.
+
+- **Database:** User profiles and scheduling data are stored in a secure database partition unique to your organization.
+
+- **Source connectors:** Agenda can be connected to a wide variety of external scheduling and meeting applications, such as Google Calendar, Outlook, Slack, or Zoom. Meeting data is then streamed from the connected source into Agenda to be processed and used to update user availability. Agenda can also be configured to send updated meeting information back to selected sources.
```

Comment:
I think it should be "..._and_ Zoom" instead of "or", so we're implying that one could connect any combination of these sources to the app, rather than just picking one of several options.